src/app/modules/users/users-list/users-list.component.scss

```diff
@@ -8,32 +8,88 @@ table {
     color: black;
   }
 
-  .example-container {
-    display: flex;
-    flex-direction: column;
-    min-width: 300px;
-  }
-  
-  .example-header {
-    min-height: 64px;
-    padding: 8px 24px 0;
-    max-width: 25%;
-  }
-  
-  .mat-form-field {
+  .custom-form-field {
     font-size: 14px;
     width: 100%;
   }
   
-  .mat-table {
-    overflow: auto;
-    max-height: 500px;
-  }
-
   .panel {
       cursor: pointer;
   }
-  .tool-bar{
-      background-color: $primary-color;
+
+  th.mat-header-cell {
+    border-bottom-style: none;
+  }
+  td.mat-cell {
+    border-bottom-style: none;
+  }
+
+ .c-toolbar {
+    color: $primary-color;
+    font-size: initial;
+    display: flex;
+    align-items: center;
+    background: whitesmoke;
+    justify-content: space-between;
+  }
+  .mat-button {
+    font-weight: 600;
+  }
+  .home {
+    cursor: pointer;
+    padding-top: 8px;
+  }
+  .pageName {
+    background-color: white;
+    padding: 6px 10px 4px 25px;
+    color: $primary-color;
   }
-  
\ No newline at end of file
+
+  .actions{
+    color: $primary-color;
+    background-color: #f3f4f5;
+  }
+
+  .mat-header-cell{
+    color: black;
+    font-weight: 600;
+  }
+  .message {
+    text-align: center;
+    background-color: white;
+    height: 500px;
+    padding-top: 15%;
+  }
+  .mat-table {
+    overflow: auto;
+    height: 550px;
+  }
+  
+  .custom-table-container {
+    position: relative;
+    overflow: auto;
+  }
+
+  .mat-spinner {
```

Comment:
don't override the default classes. Use themes instead

Reply:
Removed